src/python/TaskWorker/Worker.py

```diff
@@ -97,6 +97,14 @@ def processWorkerLoop(inputs, results, resthost, resturi, procnum, logger):
                     logger.exception('Traceback follows:')
         t1 = time.time()
         logger.debug("%s: ...work on %s completed in %d seconds: %s", procName, task['tm_taskname'], t1-t0, outputs)
+
+        try:
+            out, _, _ = executeCommand("ps u -p %s | awk '{sum=sum+$6}; END {print sum/1024}'" % os.getpid())
```

Comment:
I would not add it ad-hoc for this test, but if it becomes part of COMP environment (as it seems the case since it is available in CRAB Server) it is worth to into the direction of using it, may it means to encourage http group to add it to the standard VM environment we work upon.